### DISCUSSION TITLE
I can no longer export my app

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#11224] [Help] [answered]
![image](https://github.com/Budibase/budibase/assets/308646/141b7941-6b65-4b1f-9cd1-f9573a0ba819)

![image](https://github.com/Budibase/budibase/assets/308646/7962c7c4-4e1a-4344-b111-ee98675759ba)

The UI for this seems to have changed, and when I take the option to export my edited app I just get this error.  No other clues :(

**Comment by ConorWebb96** [comment, marked as answer]
Hey @kpturner,

Did this error occur when you were trying to export a encrypted version? If so I believe this to be a bug as I've just tried it myself. If you try to export an app without the encryption toggle on, it seems to export the app just fine.

I'm going to write up a bug report for the encrypted version and well hopefully get it fixed swiftly. i'll edit this message whenever I've written the bug report to include the link to the issue raised.

Issue can be found here: https://github.com/Budibase/budibase/issues/11227.

I hope this helps.

**Reply by Abd-Atif** [reply]
![image](https://github.com/Budibase/budibase/assets/94058099/815ad33c-339e-42e1-8d89-0572f52896ca)

![image](https://github.com/Budibase/budibase/assets/94058099/1d45ec11-f5b1-4599-8723-885530a99447)

The app is currently using version 2.21.8.
unable to export, tried both approaches. kindly help !!

**Comment by kpturner** [comment]
Unencrypted works thanks
